setup.py

```diff
@@ -43,9 +43,8 @@ def get_version():
     classifiers=[
         'Development Status :: 5 - Production/Stable',
 
-        'Intended Audience :: Religion',
-        'Topic :: Religion',
-
+        'Intended Audience :: Other Audience',
```

Comment:
This change introduces backward compatibility issues with the previous versions of the package.

Most of our users are using it in a strictly religious way. I do not believe that removing that from them would be a good choice.